.github/workflows/next-release.yml

```
@@ -30,7 +30,7 @@ jobs:
       - name: Installation && Build SDK
         run: yarn --frozen-lockfile && yarn bootstrap-ci
       - name: Lint
-        run: yarn lerna-workspaces run lint
+        run: yarn lerna-workspaces run lint && cd docusaurus && npx prettier --check '**/*.mdx'
```

```suggestion
        run: yarn lerna-workspaces run lint && (cd docusaurus; npx prettier --check '**/*.mdx')
```

For the future, running this kind of thing in a subprocess can be nice to not affect the rest of the execution so the next step would have to cd back up to root manually. In parentheses, it runs in a subprocess and isn't moving the main process anywhere